packages/react-components/src/components/Button/Button.tsx

```diff
@@ -62,12 +71,16 @@ export const Button = React.forwardRef<
       children,
       href,
       onClick,
+      animatedLabel,
       ...props
     },
     ref
   ) => {
+    const [labelWidth, setLabelWidth] = React.useState<number>(0);
+    const [isLabelOpen, setIsLabelOpen] = React.useState<boolean>(false);
```

Comment:
There is no need to pass types if you providing the initial values to the useState.